src/index.tsx

```diff
@@ -29,6 +29,8 @@ export {
   ScreenStackHeaderSearchBarView,
 } from './components/ScreenStackHeaderConfig';
 
+// NativeModules are not imported anywhere, but need to stay here so the TurboModule is registered
+export { default as NativeModules } from './fabric/NativeScreensModule';
```

Comment:
So this ended up being necessary as I've suggested, or this is just a left over? 

If this is needed - we don't want this in public API. It should be enough to import it in some other module loaded on any import from `react-native-screens/`